traffic_ops/traffic_ops_golang/asn/asns.go

```diff
@@ -340,14 +340,26 @@ func Update(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// check if asn already exists
```

Comment:
Code looks good. D you mind adding a quick test for this functionality please?
Basically add an integration test that `GET`s the ASNs, then take the ASN ID from that ASN and try to create a new ASN using that same ID, and expect an error.